src/components/Button/Button.js

```diff
@@ -1,14 +1,20 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './Button.css';
 
 class Button extends React.Component {
   render() {
     return (
-      <button className={`p-button--${this.props.type}`}>{this.props.value}</button>
-    )
+      <button className={`p-button--${this.props.modifier}`}>{this.props.value}</button>
```

Comment:
If there is no modifier does this return `p-button--`?

Reply:
Good catch @anthonydillon, completely missed that

Reply:
`defaultProps` addresses this - if you check the `App.js`, base button has no modifier prop.

`Button.defaultProps = { modifier: 'base' };`

Reply:
You're right @barrymcgee. That means I didn't miss it 😎

Reply:
Never doubted you @Lukewh 👊🏻